lib/spack/spack/build_systems/python.py

```diff
@@ -409,13 +433,13 @@ class PythonPipBuilder(BaseBuilder):
     legacy_long_methods = ("install_options", "global_options", "config_settings")
 
     #: Names associated with package attributes in the old build-system format
-    legacy_attributes = ("build_directory", "install_time_test_callbacks")
+    legacy_attributes = ("archive_files", "build_directory", "install_time_test_callbacks")
```

Comment:
Not used by the base class, but used by numpy (soon)/scipy/matplotlib

Reply:
OT: At some point we should change this attribute name to `forwarded_attribute`. I called that "legacy" because when the Pr was merged we agreed to remove single class packages with time, but we have changed our mind since then and this name makes little sense now that it might need updates.